packages/design/src/Indicator/Indicator.jsx

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 import React from 'react';
 import styled from 'styled-components';
+import { alignItems } from 'design/system';
```

Comment:
I can revert this if you want. I just thought it would be nice to have loaders spinning in our forms next to the submit/cancel buttons in processing mode. `alignItem` was set so that we can align it centered flexed next to buttons